src/pysdmx/model/concept.py

```diff
@@ -181,28 +179,7 @@ class ConceptScheme(Struct, frozen=True, omit_defaults=True):
         description: Additional descriptive information about the scheme
             (e.g. "The set of concepts in the SDMX Glossary").
         version: The scheme version (e.g. 2.0)
-        concepts: The list of concepts in the scheme.
+        items: The list of concepts in the scheme.
     """
 
-    id: str
-    name: str
-    agency: str
-    description: Optional[str] = None
-    version: str = "1.0"
-    concepts: Sequence[Concept] = ()
-
-    def __iter__(self) -> Iterator[Concept]:
-        """Return an iterator over the list of concepts."""
-        yield from self.concepts
-
-    def __len__(self) -> int:
-        """Return the number of concepts in the concept scheme."""
-        return len(self.concepts)
-
-    def __getitem__(self, id_: str) -> Optional[Concept]:
-        """Return the concept identified by the given ID."""
-        out = list(filter(lambda concept: concept.id == id_, self.concepts))
-        if len(out) == 0:
-            return None
-        else:
-            return out[0]
+    items: Sequence[Concept] = ()
```

Comment:
As in Codelists, my preference would be to use concepts.

Reply:
Yes, please check latest commit that solves this, this is outdated code